Directory.Packages.props

```diff
@@ -170,10 +170,12 @@
     <PackageVersion Include="Microsoft.Orleans.Server" Version="8.2.0" />
     <PackageVersion Include="Microsoft.Orleans.Sdk" Version="8.2.0" />
     <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
-    <PackageVersion Include="System.Text.Json" Version="8.0.4" />
-    <PackageVersion Include="Azure.Core" Version="1.42.0" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.5" />
+    <PackageVersion Include="Azure.Core" Version="1.44.1" />
     <PackageVersion Include="Azure.Identity" Version="1.12.0" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" /> <!-- https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3313 -->
     <PackageVersion Include="System.Formats.Asn1" Version="8.0.1" />
+    <PackageVersion Include="System.Net.Http" Version="4.3.4" />
+    <PackageVersion Include="System.Text.RegularExpressions" Version="4.3.1" />
```

Comment:
These shouldn't be necessary. Why are they being added?

Reply:
I turned on the nuget audit temporarily on this branch before pushing to make sure we didn't have any vulnerable transitive dependencies, and it was complaining about these two.

Reply:
We should find where those 2 are coming from and fix the real location. No one should be depending on these packages anymore.

Reply:
I've removed these lines because we shouldn't be making this change, which could affect our product assets.

I dug into this and the only things that are pulling in old versions of these packages are tests. and that is because we are using a super old xunit.assert version - 2.4.2 - over 2 years old.

We should just update our xunit.assert, but not in this PR. I will open one for main.